packages/contracts/src/dollar/core/CreditNFT.sol

```diff
@@ -55,30 +56,30 @@ contract CreditNFT is ERC1155Ubiquity {
 
         //insert new relevant block number if it doesn't exist in our list
         // (linked list implementation won't insert if dupe)
-        _sortedBlockNumbers.pushBack(expiryBlockNumber);
+        require(_sortedBlockNumbers.pushBack(expiryBlockNumber));
 
         //update the total supply for that expiry and total outstanding debt
         _tokenSupplies[expiryBlockNumber] =
             _tokenSupplies[expiryBlockNumber] + (amount);
         _totalOutstandingDebt = _totalOutstandingDebt + (amount);
     }
 
-    /// @notice Burn an amount of Credit NFT expiring at a certain block from
+    /// @notice Burn an amount of creditNFTs expiring at a certain block from
```

Comment:
Same as earlier comment.

Reply:
Changed to `CreditNFT` as that is the contract name